frontend/components/site/SiteSettings/AdvancedSiteSettings.jsx

```diff
@@ -34,105 +34,90 @@ export const AdvancedSiteSettings = ({
   handleSubmit,
 }) => (
   <form className="settings-form settings-form-advanced" onSubmit={handleSubmit}>
-    <div className="usa-grid">
-      <div className="usa-width-one-whole">
-        <label htmlFor="engine">Static site engine</label>
-        <Field
-          name="engine"
-          id="engine"
-          component={p =>
-            <SelectSiteEngine
-              value={p.input.value}
-              onChange={p.input.onChange}
-              className="form-control"
-            />
-          }
-        />
-      </div>
-    </div>
-
-    {/* CUSTOM CONFIG */}
-    <div className="usa-grid">
-      <div className="usa-width-one-whole">
-        <div className="well">
-          <h3 className="well-heading">Site configuration</h3>
-          <p className="well-text">
-            Add additional configuration in yaml to be added to your
-            {' '}
-            <code>_config.yml</code> file when we build your site&apos;s primary branch.
-          </p>
-          <Field
-            component="textarea"
-            name="config"
-            className="form-control"
-          />
-        </div>
-      </div>
-    </div>
-
-    {/* DEMO CONFIG */}
-    <div className="usa-grid">
-      <div className="usa-width-one-whole">
-        <div className="well">
-          <h3 className="well-heading">Demo configuration</h3>
-          <p className="well-text">
-            Add additional configuration in yaml to be added to your
-            {' '}
-            <code>_config.yml</code> file when we build your site&apos;s demo branch.
-          </p>
-          <Field
-            component="textarea"
-            name="demoConfig"
-            className="form-control"
-          />
-        </div>
-      </div>
-    </div>
-
-    {/* PREVIEW CONFIG */}
-    <div className="usa-grid">
-      <div className="usa-width-one-whole">
-        <div className="well">
-          <h3 className="well-heading">Preview configuration</h3>
-          <p className="well-text">
-            Add additional configuration in yaml to be added to your
-            {' '}
-            <code>_config.yml</code> file when we build a preview branch for your site.
-          </p>
-          <Field
-            component="textarea"
-            name="previewConfig"
+    <div className="well">
+      <label htmlFor="engine">Static site engine</label>
```

Comment:
What do you think about increasing the font weight of the form element `label`s? I personally like them a little heavier.

Reply:
I'm totally OK with adding that back in. I was leaning on the USWDS for guidance.